### PR TITLE
Added Season scene exception indicators to displayShow

### PR DESCRIPTION
**Pull request by p0psicles**
This PR is here for everyone to review and give his/here approval.
If things are missing, we need to add features or whatever, let me know. Will push against this PR.

If approved i'll merge.

, using the scene_exceptions saved in the cache scene_exceptions table.

Xem season scene mappings are shown through the xem icon, through the img elements title attribute. Sickrage's scene name exceptions are indicated through the sickrage ico file.

We do need to change the sickrage logo to the Medusa logo.

Xem scene exceptions are mapped from the season to sceneseason json.
Season excepions from the sickrage github file are directly mapped to the season number.

https://github.com/pymedusa/SickRage/issues/50

### CODE REVIEW

**Comment by medariox** [comment]
```
2016-03-03 15:41:40 Thread-18 :: [f98aec4] Failed doing webui callback: Traceback (most recent call last):
  File "/home/orangepi/.sickrage-exc/sickbeard/webserve.py", line 287, in async_call
    result = function(**kwargs)
TypeError: getSeasonSceneExceptions() takes exactly 3 arguments (2 given)

2016-03-03 15:41:12 Thread-16 :: [f98aec4] Failed doing webui callback: Traceback (most recent call last):
  File "/home/orangepi/.sickrage-exc/sickbeard/webserve.py", line 287, in async_call
    result = function(**kwargs)
TypeError: getSeasonSceneExceptions() takes exactly 3 arguments (2 given)

2016-03-03 15:40:25 Thread-20 :: [f98aec4] Failed doing webui callback: Traceback (most recent call last):
  File "/home/orangepi/.sickrage-exc/sickbeard/webserve.py", line 287, in async_call
    result = function(**kwargs)
TypeError: getSeasonSceneExceptions() takes exactly 3 arguments (1 given)

2016-03-03 15:39:25 Thread-19 :: [f98aec4] Failed doing webui callback: Traceback (most recent call last):
  File "/home/orangepi/.sickrage-exc/sickbeard/webserve.py", line 287, in async_call
    result = function(**kwargs)
TypeError: getSeasonSceneExceptions() takes exactly 3 arguments (2 given)
```
Every time I open a show page I get this error.

File: sickbeard/webserve.py [hunk 1]
```diff
@@ -1224,6 +1224,25 @@ def getDBcompare():
...
+        """
+
+        exceptionsList = {}
+        exceptionsList['seasonExceptions'] = sickbeard.scene_exceptions.get_all_scene_exceptions(indexer_id)
+
+        xem_numbering_season = {}
+        for tvdb_season_ep, anidb_season_ep in get_xem_numbering_for_show(indexer_id, indexer).iteritems():
+            if not xem_numbering_season.get(str(tvdb_season_ep[0])):
+                xem_numbering_season[str(tvdb_season_ep[0])] = str(anidb_season_ep[0])
+
```
Comment (medariox) [inline sickbeard/webserve.py]: Why not just?
```
for season, xem_season in get_xem_numbering_for_show(indexer_id, indexer).items():
            xem_numbering_season[season[0]] = xem_season[0]
```
.items() for py3 compatibility. Don't need str() and I would avoid it everywhere it's not needed.

Comment (labrys) [inline sickbeard/webserve.py]: @medariox and even better yet, make it a dict comprehension:
```python
xem_numbering_season = {
    season[0]: xem[0] 
    for season, xem in get_xem_numbering_for_show(indexer_id, indexer).items()
}
```
also exceptionLists should be both snake_case (and keys shouldn't be camel case) and a dict literal  e.g.:
```python
exception_lists = {
    'season_exceptions': sickbeard.scene_exceptions.get_all_scene_exceptions(indexer_id),
    'xem_numbering': xem_numbering_season,  # can also use dict comprehension here directly.
}

Comment (p0psicles) [inline sickbeard/webserve.py]: Items() is slow and memory expensive in python2. Actually i Used items() but changed it after feedback. I need the string there else its passed as int. If someone wants to commit the snakecase, i think it's fine.

**Comment by medariox** [comment]
Approved

**Pull request merged**